drivers/usb/device/Kconfig

```diff
@@ -21,7 +21,7 @@ config USB_DW_IRQ_PRI
 	help
 	USB interrupt priority.
 
-config SYS_LOG_USB_DW_LEVEL
+config SYS_LOG_USB_DRIVER_LEVEL
 	int
 	prompt "DesignWare USB driver log level"
```

Comment:
thx, it also no longer depends on USB_DW. btw, does it make sense to move the option to drivers/usb/Kconfig? Then it can be used for device and host drivers.